mainnet/collections.json

```diff
@@ -5,5 +5,12 @@
     "power": 2,
     "name": "ETHUSD",
     "tolerance": 500000
+  },
+  {
+    "jobIDs": [11, 12, 13, 14, 15],
+    "aggregationMethod": 2,
+    "power": 2,
```

Comment:
If you update power for all SKALE jobs to `6`, you need to update the power for collection to `6` as well
@manuelbarbas